src/main/kotlin/uk/gov/gdx/datashare/service/DataReceiverService.kt

```diff
@@ -71,6 +73,8 @@ class DataReceiverService(
       publisher.name,
     )
 
+    meterRegistry.counter("EVENT_ACTION.IngressEventPublished", "eventType", eventPayload.eventType)
```

Comment:
If so, perhaps this would make sense as 
```kotlin
private val ingressEventPublishedCounter by lazy { meterRegistry.counter("EVENT_ACTION.IngressEventPublished", "eventType", eventPayload.eventType) }
```
?

Reply:
We did have them pulled out as variables at the top, but in some files they were starting to stack up quite a lot and only used once, so Olly thought we might as well initialise them in the method they're used

Reply:
Also yes, `.increment()` should definitely be there 🙃